test/app-luatest/uri_test.lua

```diff
@@ -0,0 +1,56 @@
+local uri = require("uri")
+local t = require("luatest")
+
+local g = t.group("uri_integration")
+
+local uri_encoding_cases = {
+    { url = "a", url_encoded = "%61" },
```

Comment:
It is the opposite to what I expect. Latin letter 'a' shouldn't be escaped. Russian letters (below) should.